docs/docs/cmd/teams/channel/channel-remove.md

```diff
@@ -16,8 +16,11 @@ m365 teams channel remove [options]
 `-n, --name [name]`
 : The name of the channel to remove. Specify id or name but not both
 
-`-i, --teamId <teamId>`
-: The ID of the team to which the channel to remove belongs
+`-i, --teamId [teamId]`
+: The ID of the team to which the channel to remove belongs. Specify either `teamId` or `teamName` but not both
```

Comment:
In accordance with the Spec, could you remove the short `-i` here?

Could you change the `-c` short of the `--id` option to `-i`?